tests/testthat/test-bump_version.R

```diff
@@ -1,92 +1,94 @@
 test_that("bump_version() works -- dev", {
```

Comment:
the diff here: removing the skips as I don't think we need them anymore, same for the variant, and restructuring the test files into paragraphs.